indico/modules/auth/blueprint.py

```diff
@@ -16,8 +16,10 @@
 
 from __future__ import unicode_literals
 
-from indico.modules.auth.controllers import (RHLogin, RHLoginForm, RHLogout, RHAssociateIdentity, RHRegister,
-                                             RHResetPassword)
+from flask import request
+
+from indico.modules.auth.controllers import (RHLogin, RHLoginForm, RHLogout, RHRegister, RHLinkAccount
```

Comment:
How did you not miss that missing comma, that's breaking startup ;)

:boom: :volcano: